app/components/playlist-selector.tsx

```
@@ -1,89 +1,127 @@
-          Cancel
+          <OutlinedIcon icon="add" />
+          Create New Playlist
         </button>
+        {playlists.length === 0 && !isLoading && (
+          <p className="text-center text-[--md-sys-color-on-surface-variant] py-4">
+            No playlists found. Create one to get started!
+          </p>
+        )}
+        {isLoading ? (
+          <div className="flex justify-center py-4">
+            <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-[--md-sys-color-primary]"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)